src/commands/generate.ts

```diff
@@ -48,32 +53,73 @@ const generatePrompt = (templates: string[], load: any, options: OptionValues) =
         if (/^([A-Za-z\-\_\d])+$/.test(input)) return true;
         else return 'Project name may only include letters, numbers, underscores and hashes.';
       }
+    },
+    {
+      name: 'app-confirm',
+      type: 'confirm',
+      default: false,
+      message: 'Do you want to generate template for UI app?'
```

Comment:
i would swap `app-confirm` question with `project-name` question. `project-name` should be last question that appears

Reply:
...or the first one :)

Reply:
'project-name' is first question now